src/Build/Definition/Project.cs

```diff
@@ -505,7 +507,7 @@ public static Project FromProjectRootElement(ProjectRootElement rootElement, Pro
                 options.SubToolsetVersion,
                 options.ProjectCollection ?? ProjectCollection.GlobalProjectCollection,
                 options.LoadSettings,
-                options.EvaluationContext);
+                options.EvaluationContext ?? _sharedEvaluationContext);
```

Comment:
I'm not sure I understand the implications of this. Can you describe what it helps?

Reply:
This was an experiment which was not supposed to become part of this PR. I'll remove it.

Reply:
Removed.